app/views/posts/index.html.erb

```diff
@@ -1,28 +1,17 @@
-<link href="https://fonts.googleapis.com/css?family=Tenor+Sans&display=swap" rel="stylesheet">
-<link rel="stylesheet" type="text/css" href="base.css">
-
-<div class="topnav">
-  <a class="active" href="#home">Home</a>
-  <a href="#create">Create</a>
-<a href="#friends">Friends</a>
-</div>
-
-<%= form_for @post do |form| %>
+ <%= form_for @post do |form| %>
 <%= form.label :name %>
 <%= form.text_field :username %><p>
   <%= form.label :message %>
   <%= form.text_field :message %>
```

Comment:
Please could you check the indentation here

Reply:
Done. Run through auto-indent on Atom.